src/targetWidths.js

```diff
@@ -6,12 +6,12 @@ function targetWidths() {
 
   const ensureEven = n => 2 * Math.round(n / 2);
 
-  resolutions.push(prev);
   while (prev <= MAX_SIZE) {
-    prev *= 1 + (INCREMENT_PERCENTAGE / 100) * 2;
     resolutions.push(ensureEven(prev));
+    prev *= 1 + (INCREMENT_PERCENTAGE / 100) * 2;
   }
 
+  resolutions.push(MAX_SIZE);
```

Comment:
@jayeb how do we feel about this? It doesn't abide by the 16% increment but it does feel right to include the max render size in the srcset.

Reply:
👍